src/Common/Services.fs

```diff
@@ -66,11 +66,11 @@ module String =
     /// converts a list of chars to a string
```

Comment:
note that 
`implode` is unused (dead legacy)
`explode` is an implementation of .Net's `String.ToCharArray()`